fearless/Modules/Staking/StakingMain/ViewModel/StakingStateViewModelFactory+Alerts.swift

```diff
@@ -5,7 +5,21 @@ extension StakingStateViewModelFactory {
     func stakingAlertsForNominatorState(_ state: NominatorState) -> [StakingAlert] {
         switch state.status {
         case .active:
-            return []
+            guard
```

Comment:
Redeemable alert should not depend on staking status. Because of the fact that this information is stored in StakingLedger and not in Nomination then tokens can be redeemed anytime unless it is election period.